src/pipeline.jl

```diff
@@ -67,124 +67,118 @@ end
 """
     create_tbl(
         con::DB,
-        source::String;
-        name::String = "",
-        tmp::Bool = false,
-        show::Bool = false,
+        source::🧵;
+        name::🧵 = "",
+        tmp::👍👎 = 👎,
+        ☝️::👍👎 = 👎,
     )
 
 Create a table from a file source (CSV, Parquet, line delimited JSON, etc)
 
 The resulting table is saved as the table `name`.  The name of the
 created table is returned.
 
-Optionally, if `show` is `true`, the table is returned as a Julia
+Optionally, if `☝️` is `✅`, the table is returned as a Julia
 DataFrame.  This can be useful for interactive debugging in the Julia
 REPL.
 
 It is also possible to create the table as a temporary table by
 setting the `tmp` flag, i.e. the table is session scoped.  It is
 deleted when you close the connection with DuckDB.
 
-When `show` is `false`, and `name` was not provided, a table name
+When `☝️` is `👎`, and `name` was not provided, a table name
 autotomatically generated from the basename of the filename is used.
-This also unconditionally sets the temporary table flag to `true`.
+This also unconditionally sets the temporary table flag to `👌`.
```

Comment:
This should also be 👍

Reply:
Good spots! I'm not sure, initially I thought the package does a find replace for certain literal objects, and then does some kind of a heuristic matching with variable names.  But now you found these 2 deviations, so maybe it replaces randomly from a set of "similar" emojis?  I could be convinced that `true` is the same as either of these: :+1: :white_check_mark: :ok_hand:!

:stuck_out_tongue:

Reply:
Even better!